charts/prometheus-cloudwatch-exporter/Chart.yaml

```
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: "0.12.2"
+appVersion: "0.13.0"
```

sorry, if thought we've skipped on version in the "version:" filed.
0.14.0 would be correct if tag is 0.14.0

```suggestion
appVersion: "0.14.0"
```